src/wallet/walletdb.cpp

```diff
@@ -375,6 +386,17 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
 
             pwallet->LoadKeyPool(nIndex, keypool);
         }
+        else if (strType == "pending_firstupdate")
+        {
+            std::string strName;
+            ssKey >> strName;
+
+            CNamePendingData pending;
+            ssValue >> pending;
+
+            pwallet->namePendingMap[strName] = pending;
+            LogPrintf("Loaded pending name_firstupdate %s\n", strName);
```

Comment:
I don't think we should log all of them (especially not at the default log level with `LogPrintf` - perhaps `LogPrint` with a suitable log type would be fine).

Reply:
Fixed.